### ISSUE TITLE
Audit Vercel site for parity with WP site

### ISSUE THREAD

**Issue by lianilychee** [open-austin/website-v2#52]
lianilychee (Open Austin prez / Ad Hoc product)
 [Yesterday at 7:22 PM](https://open-austin.slack.com/archives/C03A77BFGRM/p1666052543172059)
(delivery) Could you help me understand what needs to be done now in order to point [open-austin.org](http://open-austin.org/) and [www.open-austin.org](http://www.open-austin.org/) to the [vercel site](https://next-js-boilerplate-three-alpha.vercel.app/)?  I'm trying to suss out what actually can go-live-by-EOY-but-needs-iteration, vs what is actually a blocker
(edited)
7 replies

Dan
  [17 hours ago](https://open-austin.slack.com/archives/C03A77BFGRM/p1666055912409579?thread_ts=1666052543.172059&cid=C03A77BFGRM)
Basically some kind of audit that we have the parts we care about in place. I am I little concerned about losing some of the pieces, e.g. the blog posts, if we’re pla

**Comment by lianilychee** [comment]
Finished
